lib/appearance.rb

```diff
@@ -110,6 +113,10 @@ def inner_word
     #
     # Returns Nokogiri::XML object
     def rss_feed
-      Nokogiri::XML(open(url))
+      xml = Rails.cache.fetch("xml", :expires_in => 1.day) do
```

Comment:
INFO: Style/HashSyntax: Use the new Ruby 1.9 hash syntax. (https://github.com/bbatsov/ruby-style-guide#hash-literals)